### ISSUE TITLE
1.0.0

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/chalk#44]
We should get out an 1.0.0 soon.

Some tasks that needs to be done before hand (please fill in if I left out something):

- [x] #36 
- [x] #31 no idea what's going on there but I suspect it has nothing to do with us and more to do with npm
- [x] Readme improvements. Better description. The text feels a bit crude. And anything else that could make Chalk easier to get started with and use.
- [x] #46
- [x] Anything feature wise we're missing?
- [x] sindresorhus/supports-color#13
- [x] Changelog
- [x] Some manual testing with popular consumers like Bower, Gulp.

// @jbnicolai

**Comment by sindresorhus** [comment]
I'm working on the changelog now and just need to finish it and test with popular dependents. Will probably do a release tomorrow.

**Comment by jbnicolai** [comment]
Ah, was just about to pick up the changelog this afternoon. :+1: 

I'll do some manual testing with major dependents as well.

**Comment by jbnicolai** [comment]
I suggest we wait for sindresorhus/supports-color/pull/17

**Comment by sindresorhus** [comment]
Agreed

**Comment by jbnicolai** [comment]
:+1:  be sure to include the two changes to `supports-color` in the release notes. I believe other dependencies haven't seen major changes.

**Comment by jbnicolai** [comment]
I've done some testing with `gulp`, `grunt` and `karma`. All seemed to work fine :)

**Comment by sindresorhus** [comment]
Cool. Yeah. Already done. Just doing some final tweaks and testing ;)

**Comment by sindresorhus** [comment]
@jbnicolai woot! => https://github.com/sindresorhus/chalk/releases/tag/v1.0.0

*Feel free to do any tweaks or changes to the changelog.*

I'm going to wait until tomorrow to publicise this in case of some last minute bug fix.

Let's start the PRing popular projects with the 1.0.0 ;)

![687474703a2f2f7777772e7265616374696f6e676966732e636f6d2f77702d636f6e74656e742f67616c6c6572792f64616e63652d70617274792f74756d626c725f6d306f73727268714d6b31717a6376376e6f345f3235302e676966](https://cloud.githubusercontent.com/assets/170270/6324294/aa697ff6-bb6a-11e4-9219-a5cf9d874f89.gif)

**Comment by jbnicolai** [comment]
![high-five](http://www.reactiongifs.us/wp-content/uploads/2013/11/high_five_stephen_colbert.gif)

I'll proofread the changelog and start creating PRs for our biggest dependants this afternoon (I'll automate the process as I did last time and share a gist) :smile:

**Comment by sindresorhus** [comment]
Cool. Make sure to include a link to the release notes in the PRs ;)
